src/AaveV3StrategicAssets_20230606/README.md

```diff
@@ -0,0 +1,207 @@
+For them, the underlying, the sdToken address of the token, and the address where to deposit the underlying to receive sdToken.
+
+`function removeSdToken(address underlying) external onlyOwner`
+
+Removes the ability to interact with a certain sdToken.
+
+`function withdrawERC20(address token, address to, uint256 amount) external onlyOwner`
+
+Sends ERC20 tokens to an address. Withdrawal mechanism.
+
+`function setStrategicAssetsManager(address _manager) external onlyOwner`
+
+Adds manager role.
+
+`function removeStrategicAssetManager() external onlyOwner`
+
+Revokes manager role.
+
+#### VeTokenManager.sol
+
+```
+ function buyBoost(
+    address underlying,
+    address delegator,
+    address receiver,
+    uint256 amount,
+    uint256 duration
+  ) external onlyOwnerOrManager
+```
+
+Purchase boost to incentivize rewards earned by locking (up to 2.5x of earnings). Spend fee token.
+For more info see: https://doc.paladin.vote/warden-boost/boost-market
+
+The idea is to increase the yield in the provided liquidity.
+For example, pay 10 BAL to boost rewards in a veBAL pool up to 2.5x times, to earn more BAL in return.
+
+```
+ function sellBoost(
+    address underlying,
+    uint256 pricePerVote,
+    uint64 maxDuration,
+    uint64 expiryTime,
+    uint16 minPerc,
+    uint16 maxPerc,
+    bool useAdvicePrice
+  ) external onlyOwnerOrManager
+```
+
+Owner of veToken allows others to incentivize their liquidity pools by selling boost. The price can be chosen by the user, or by setting useAdvicePrice, let Warden determine the price.
+The seller of boost receives the native token.
+
+```
+  function updateBoostOffer(
+    address underlying,
+    uint256 pricePerVote,
+    uint64 maxDuration,
+    uint64 expiryTime,
+    uint16 minPerc,
+    uint16 maxPerc,
+    bool useAdvicePrice
+  ) external onlyOwnerOrManager
+```
+
+Allows the user to update an existing offer to sell boost.
+
+`function removeBoostOffer(address underlying) external onlyOwnerOrManager`
+
+Removes a boost offer.
+
+` function claim(address underlying) external onlyOwnerOrManager`
+
+Claim rewards earned by selling boost.
+
+`function setSpaceId(address underlying, bytes32 _spaceId) external onlyOwnerOrManager`
+
+Sets the spaceID that's used by protocol on Snapshot for voting. For example, "balancer.eth" is Balancer's spaceId on Snapshot.
+
+```
+ function setDelegateSnapshot(
+    address underlying,
+    address newDelegate
+  ) external onlyOwnerOrManager
+```
+
+Delegate tokens so they can vote on Snapshot.
+
+`function clearDelegateSnapshot(address underlying) external onlyOwnerOrManager`
+
+Remove the active delegate.
+
+```
+function voteAragon(
+    address underlying,
+    uint256 voteData,
+    bool support
+  ) external onlyOwnerOrManager
+```
+
+Vote on Aragon (as opposed to off-chain votes done via Snapshot). veCRV uses Aragon voting.
+
+** Since DAO is unlikely to hold veCRV immediately, this function can likely be removed initially.**
```

Comment:
DAO has some CRV, maybe worth including, paging @sakulstra for an opinion too

Reply:
Sorry, docs were outdated!